sublet/services/propertyService.ts

```diff
@@ -0,0 +1,154 @@
+import { PropertyInterface } from '../interfaces/Property';
+//import getCsrf from '../utils/csrf';
+
+function getCookie(name: string) {
+  let cookieValue = "";
+  if (document.cookie && document.cookie !== '') {
+    const cookies = document.cookie.split(';');
+    for (let i = 0; i < cookies.length; i++) {
+      const cookie = cookies[i].trim();
+      // Does this cookie string begin with the name we want?
+      if (cookie.startsWith(name + '=')) {
+        cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
+        break;
+      }
+    }
+  }
+  return cookieValue;
```

Comment:
`pref`: avoid for loops and prefer a more functional approach. 
See: https://github.com/pennlabs/penn-mobile/blob/master/frontend/utils/csrf.ts
`pref`: Also prefer f strings over using + for readability

Also this would be one of the things that should be refactored into being shared with portal (prolly also the other auth stuff too)